AutoROM/licensed/AutoROM.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+import sys
+import requests
+import warnings
+import hashlib
+import tarfile
+import pathlib
+import click
+import io
+
+from collections import namedtuple
+
+from tqdm import tqdm
+
+if sys.version_info < (3, 9):
+    import importlib_resources as resources
+else:
+    import importlib.resources as resources
+
+# simply download tar file to specified dir
+def download_tar_to_buffer(url="https://roms8.s3.us-east-2.amazonaws.com/Roms.tar.gz"):
```

Comment:
Can you delete the old code which did this, or refactor it to use the same code?

Reply:
I'm trying to figure out a way to share this code. The issue is that the subpackage gets installed at the same time as the root package so it can't import the parent package. Now you think you'd be able to adjust `sys.path` to be able to find `AutoROM` but pip creates a new temporary directory for each package that's installed so I have no idea where the parent package ends up, it's no longer in the parent directory of the subpackage.

I think the best case solution is that when building the source package we can copy these files to both packages simultaneously such as to have one version of these files that end up in both packages.